src/Fable.Cli/Entry.fs

```diff
@@ -183,6 +183,8 @@ type Runner =
     do!
         if outDirLast = Naming.fableHiddenDir then
             Error($"{Naming.fableHiddenDir} is a reserved directory, please use another output directory")
+        elif outDirLast = "obj" then
```

Comment:
Would you prefer a link to the issue as a comment inline to explain to future visitors why this is the case, or do we rely on blame?

Reply:
It should be fine, `obj` is usually known as a special directory in .NET so it's understandable we reserve it. Now I just hope that people don't pick `.git` as out directory or similar :) Thanks!